app/src/main/res/values-de/strings.xml

```diff
@@ -747,7 +748,10 @@ Signatur: %5$s</string>
     <string name="student_financial_aid">Studienfinanzierung und Beiträge</string>
     <string name="released">Veröffentlicht</string>
 
-    <string name="new_items_format_string">%d neue Meldungen</string>
+    <plurals name="new_items_format_string">
+        <item quantity="one">%d neue Meldung</item>
```

Comment:
nit: If this is the only option, I always feel like this could be more obvious to future maintainers

```suggestion
        <item quantity="one">1 neue Meldung</item>
```